### ISSUE TITLE
Add posibility to run puppeteer with --no-sandbox

### ISSUE THREAD

**Issue by Hector68** [magesuite/magepack#6]
I got error

Failed to launch the browser process!                                                                 09:35:52
[0407/093552.508440:ERROR:zygote_host_impl_linux.cc(89)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.

TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md

  [0407/093552.508440:ERROR:zygote_host_impl_linux.cc(89)] Running as root without --no-sandbox is not supported. See https:crbug.com/638180.

  TROUBLESHOOTING: https:github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md

  at onClose (/node_modules/puppeteer/lib/Launcher.js:750:14)
  at Interface.<anonymous> (/node_modules/puppeteer/lib/Launcher.js:739:50)
  at Interface.emit (events.js:323:22)
  at Interface.close (readline.js:409:8)
  at Socket.onend (readline.js:187:10)
  at Socket.emit (events.js:323:22)
  at endReadableNT (_stream_readable.js:1204:12)
  at processTicksAndRejections (internal/process/task_queues.js:84:21)

**Comment by krzksz** [comment]
Hey @Hector68, thanks for raising the issue.
I will check if there are any downsides in always adding `--no-sandbox` so we don't need another option to be added. On the other hand, are you sure you need to run magepack as root user?

**Comment by Hector68** [comment]
I use docker container. But if I run  in another user I will get  error

```
 ERROR  Failed to launch the browser process!                                                                 12:24:07
[0407/122406.949479:FATAL:zygote_host_impl_linux.cc(116)] No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.
#0 0x556d8f89e219 base::debug::CollectStackTrace()
#1 0x556d8f802363 base::debug::StackTrace::StackTrace()
#2 0x556d8f814195 logging::LogMessage::~LogMessage()
#3 0x556d9109000e service_manager::ZygoteHostImpl::Init()
#4 0x556d8f40bf27 content::ContentMainRunnerImpl::Initialize()
#5 0x556d8f45aafa service_manager::Main()
#6 0x556d8f40a501 content::ContentMain()
#7 0x556d8f4598f5 headless::(anonymous namespace)::RunContentMain()
#8 0x556d8f45959d headless::HeadlessShellMain()
#9 0x556d8cfceaa7 ChromeMain
#10 0x7f5f6ffe72e1 __libc_start_main
#11 0x556d8cfce8ea _start
```

**Comment by vbuck** [comment]
Can confirm that this solves the same issue:
* https://github.com/vbuck/magepack/blob/feature/chrome-no-sandbox-flag/lib/generate.js#L13

Since this is a tool intended for use with build pipelines, I see no harm in disabling sandbox functionality. If you serve an application from source code that is infected with XSS or CSRF content which could exploit the the host, then you have a larger problem than to worry about!

**Comment by krzksz** [comment]
Released in 2.2.1
